js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-plusplus */
+function alternatingCharacters(arr) {
+  const deleteCounts = new Array(arr.length).fill(0);
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      if (arr[i][j] === arr[i][j + 1]) {
```

Comment:
Hey, Imran looks like a good implementation. What are your thoughts on making it more declarative? 
For example, can we use reduce and map functions of the array? 
Instead of using the name arr to represent an array of strings. Anything more descriptive like arrayOfStrings etc.
Instead of arr[i][j] and arr[i][j+1]  any meaningful names like previousCharacter/nextCharacter etc